servicetalk-concurrent-api/src/testFixtures/java/io/servicetalk/concurrent/api/SequentialPublisherSubscriberFunction.java

```diff
@@ -41,6 +43,7 @@ public Subscriber<? super T> apply(final Subscriber<? super T> subscriber) {
             throw new IllegalStateException("Duplicate subscriber: " + subscriber);
         }
         this.subscriber = subscriber;
+        numberOfSubscribers.incrementAndGet();
```

Comment:
we only increment, do we ever need to decrement this counter too (i.e. on unsubscribe?)

Reply:
In a sequential mode there is only one subscriber allowed at a time and this can be verified by `isSubscribed()` method. The goal of this feature is to count how many subscribers it saw during the lifetime of the publisher. Maybe it needs a better name. I will try `numberOfSubscribersSeen()`. LMK if you have other ideas.